library/blivet.py

```diff
@@ -146,6 +144,55 @@ def find_duplicate_names(dicts):
 
     return duplicates
 
+def new_mdarray(blivet, members, param_dict, fmt, raid_name=""):
```

Comment:
This should go in `BlivetBase`, which I made for exactly this type of application.

Reply:
Fixed: Moved function to `BlivetBase` class, removed parameters obtainable from class.